db/migrate/20160324174132_create_deployments.rb

```diff
@@ -0,0 +1,24 @@
+class CreateDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
+      t.string   :deployment_version
+      t.boolean  :containerized, :default => true
+      t.string   :deployment_method
+      t.string   :ssh_private
+      t.string   :ssh_public
+      t.string   :ssh_user
+      t.string   :rhsm_user
+      t.string   :rhsm_pass
+      t.string   :rhsm_sku
+      t.string   :rhsm_pool_id
+      t.string   :rhsm_server
+      t.string   :rhsm_activation_key
```

Comment:
I could see a case where they might want an activation key per "role" (master/node in particular), but not where they would want to have it separated on a per-host basis. Either way, I think limiting it to the deployment is the best bet currently.